src/haddock/libs/libsubprocess.py

```diff
@@ -55,6 +56,67 @@ def make_cmd(self):
         return
 
 
+class CapriJob:
```

Comment:
This class is very specific for the `caprieval` module. Could you move it to a new file in `caprieval/caprijob.py` ? the `libs` files are for very general implementations.

Reply:
Yes! I thought we wanted to keep all the "Job"-like classes in the same place. I'll create the separate file.